recipes/h2mm-c/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "h2mm-c" %}
+{% set version = "1.0.5" %}
+{% set sha256 = "949ab32a3255d286fbb09b2bb9918bac21d4fd6c0cb4216dfc264bf82fa827ce" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/harripd/H2MMpythonlib/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  script:
+    - export SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib("c") }}
```

```suggestion
    - {{ stdlib('c') }}
    - python                                 # [build_platform != target_platform]
    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
    - cython                                 # [build_platform != target_platform]
    - numpy                                  # [build_platform != target_platform]
```

see other packages that enable cross compilation. this will become important for osx-arm and aarch.
https://github.com/conda-forge/imagecodecs-feedstock/blob/main/recipe/meta.yaml#L20